tests/utils/utils.go

```diff
@@ -944,3 +947,81 @@ func SendExampleCrossChainMessageAndVerify(
 		Expect(currMessage).ShouldNot(Equal(message))
 	}
 }
+
+func InitChainConfig(
+	networkID uint32,
+	subnet interfaces.SubnetTestInfo,
+	teleporterAddress common.Address,
+) (*avalancheWarp.UnsignedMessage, string) {
+	unsignedMessage := CreateOffChainRegistryMessage(networkID, subnet, teleporterregistry.ProtocolRegistryEntry{
+		Version:         big.NewInt(2),
```

Comment:
can this be a parameter for utils function

Reply:
For sure